### ISSUE TITLE
Value to write for turn off

### ISSUE THREAD

**Issue by ltorok** [manekinekko/cafy#7]
Hi,

thanks for great job. I have one question only.
What is value to write for turn off? I can nowhere it find.

Thanks.

**Comment by manekinekko** [comment]
Hey, thanks for your support.
I still couldn't figure out the value to turn off the coffee machine. Once I find it (if it's possible), I will add it to the readme.

/related #6

**Comment by AchimHome** [comment]
Finally - I found the switch-off code by trying around.

`0d 07 84 0f 01 01 00 41`

`[0x0d, 0x07, 0x84, 0x0f, 0x01, 0x01, 0x00, 0x41]`

I suppose it's like this:
```
0d (Carriage Return) as initiation sequence
07: length of packet in bytes (from byte 1-7)
840f: trigger I/O port
01: turn off (02: turn on)
01: I/O port number
0041: checksum (from byte 0-7)
```

**Comment by manekinekko** [comment]
You rock!  Thank you 👍🏽

**Comment by ltorok** [comment]
@AchimHome Super job. Perfect. Thank You very much.

**Comment by mmastrac** [comment]
Author of `longshot`, was linked to this via a third party 😄 

For anyone looking for a turn-off command for the Delonghi Dinamica Plus, this unfortunately doesn't work. I've done my own search of the "app control" command space (this appears to be what the app calls it), and none of the values seem to trigger a power-off.

To correct the decoding slightly: command 0x84 is named "App Control" internally by Delonghi, as far as I can tell, 0x0f is a command that requests a status response (monitor commands use this). The two bytes might be "value" and "target", but I/O port may also be correct (I don't think I've seen them name them). 

If there's any other information we can exchange about the protocol, please let me know. I've gotten pretty far in decoding recipes, brew commands (incl custom), statistics and parameters. And if anyone ever figures out how to turn this Dinamica Plus off, I'm desperate!

**Comment by AchimHome** [comment]
At least the switch-off works with my older DeLonghi ELITE 650.85.

**Comment by ltorok** [comment]
Yes, for me works with **DeLonghi PrimaDonna Elite ECAM 650.85.MS**.
